<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mi Paint</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Resources/paleta-de-pintura.png</normaloff>:/Resources/paleta-de-pintura.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEstilo">
    <property name="title">
     <string>Estilo</string>
    </property>
    <addaction name="actionColor"/>
    <addaction name="actionAncho"/>
   </widget>
   <widget class="QMenu" name="menuForma">
    <property name="title">
     <string>Forma</string>
    </property>
    <addaction name="actionLineas"/>
    <addaction name="actionLibre"/>
    <addaction name="actionRectangulos"/>
    <addaction name="actionCirculos"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEstilo"/>
   <addaction name="menuForma"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/disquete.png</normaloff>:/Resources/disquete.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/add-file.png</normaloff>:/Resources/add-file.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/logout.png</normaloff>:/Resources/logout.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/pantone.png</normaloff>:/Resources/pantone.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionAncho">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/talla.png</normaloff>:/Resources/talla.png</iconset>
   </property>
   <property name="text">
    <string>Ancho</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionLineas">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/eliminar.png</normaloff>:/Resources/eliminar.png</iconset>
   </property>
   <property name="text">
    <string>Lineas</string>
   </property>
  </action>
  <action name="actionLibre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Libre</string>
   </property>
  </action>
  <action name="actionRectangulos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/rectangulo.png</normaloff>:/Resources/rectangulo.png</iconset>
   </property>
   <property name="text">
    <string>Rectangulos</string>
   </property>
  </action>
  <action name="actionCirculos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/dry-clean.png</normaloff>:/Resources/dry-clean.png</iconset>
   </property>
   <property name="text">
    <string>Circulos</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
